LICENSE

```diff
@@ -1,7 +1,8 @@
 MIT License
 
 Copyright (c) 2021 Philip May
-Copyright (c) 2021 Philip May, Deutsche Telekom AG
+Copyright (c) 2021 Timothy Wolff-Piggott
+Copyright (c) 2021 Philip May, Timothy Wolff-Piggott, Deutsche Telekom AG
```

Comment:
I think better keep one person per line.

Reply:
Should I update to the following:

```
Copyright (c) 2021 Philip May
Copyright (c) 2021 Philip May, Deutsche Telekom AG
Copyright (c) 2021 Timothy Wolff-Piggott
```

Are the two lines with your name distinguishing between your personal contributions and your contributions through project work for Deutsche Telekom AG?

Reply:
> Should I update to the following:

Yes please

> Are the two lines with your name distinguishing between your personal contributions and your contributions through project work for Deutsche Telekom AG?

Yes. Exactly.